app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -265,6 +267,29 @@ a.inplace-editing--trigger-link,
     &.edit-strategy-textarea
       margin-top: auto
 
+// Explicit styles for comment / input-esque trigger appearance
+.work-packages--activity--add-comment
+  margin: 20px 0
+
+  .inplace-editing--container
+    border: 1px solid #eee
+
+  .inplace-edit--read-value
+    padding: 10px 5px
+
+  .inplace-editing--trigger-link
+    color: #888
+    font-size: 1rem
```

Comment:
What do you mean with 'show the form fields'? And shouldn't Description and Comment look alike since they are both text areas and Watchers a select field?